src/components/v5/frame/ColonyHome/partials/ReputationChart/utils.ts

```diff
@@ -78,6 +79,18 @@ export const getTeamReputationChartData = (
       };
     });
 
+  if (topTeams.length <= WIDGET_TEAM_LIMIT) {
+    const adjustedValues = adjustPercentagesTo100(
+      topTeams.map((team) => team.value),
+      // getNormalisedDomainReputationPercentage rounds to two decimals
+      2,
+    );
+    topTeams = topTeams.map((team, idx) => ({
+      ...team,
+      value: adjustedValues[idx],
+    }));
+  }
+
```

Comment:
For the teams, again a similar situation will happen. The max length of `topTeams` will be `WIDGET_TEAM_LIMIT`, so we'll end up normalising the reputation for them to add up to `100%`.
However for the teams, the way it's implemented right now, the remaining reputation is determined by adding up the `topTeams` reputation and subtracting it from `100`.
By calling `adjustPercentagesTo100`, the remaining reputation will always be `0`.

So, I believe the best solution will be to refactor the computation for the remaining reputation (only for consistency purposes)
```
const reputationInOtherTeams = domainsWithoutRoot
    .filter((domain) => !!domain.reputationPercentage)
    .slice(WIDGET_TEAM_LIMIT)
    .reduce((reputation, team) => reputation + Number(team.reputation), 0);
```
and then apply `adjustPercentagesTo100` to the entire returned value